x/leverage/keeper/msg_server.go

```diff
@@ -187,21 +187,26 @@ func (s msgServer) RepayAsset(
 		return nil, err
 	}
 
-	if err := s.keeper.RepayAsset(ctx, borrowerAddr, msg.Amount); err != nil {
+	repaid, err := s.keeper.RepayAsset(ctx, borrowerAddr, msg.Amount)
```

Comment:
```suggestion
	successfullyRepaid, err := s.keeper.RepayAsset(ctx, borrowerAddr, msg.Amount)
```

Reply:
I think `repaid` should be sufficient in this case

Reply:
yeah `repaid` should be fine to convey context.